monitor/core/utils/__init__.py

```diff
@@ -0,0 +1,480 @@
+import hashlib
```

Comment:
copied from backend. we need to cleanup unused functions from here and from the backend util as well

Reply:
correct. We need to check which functions are redundant here and the backend.